tools/commissioner_thci/commissioner_impl.py

```diff
@@ -401,6 +404,34 @@ def getCOM_TOK(self):
         result = self._execute_and_check('token print')
         return self._hex_to_bytes(result[0])
 
+    def getCommissioningLogs(self):
```

Comment:
`getCommissioningLogs` needs to wait for the commissioning process to finish. How do you make sure of that?

Reply:
I think we have no way to control when the method is called. The caller must know when to call this method.

Reply:
Not sure if that's true of TestHarness. We don't know how `getCommissioningLogs` is used by TestHarness.